app/models/character.rb

```diff
@@ -0,0 +1,4 @@
+class Character < ActiveRecord::Base
+  has_many :skills
+  has_many :talents
+end
```

Comment:
Character is going to have Character_skills & character_talents on a 1-1 basis. Character_skills & character_talents will have a 1 to many relationship with skills & talents so I think there will be an extra step here if I'm thinking correctly.